src/driver/sqljs/SqljsDriver.ts

```diff
@@ -124,8 +124,9 @@ export class SqljsDriver extends AbstractSqliteDriver {
             }
         }
         else {
-            const content = JSON.stringify(this.databaseConnection.export());
-            PlatformTools.getGlobalVariable().localStorage.setItem(path, content);
+            const database: Uint8Array = this.databaseConnection.export();
+            const databaseArray = [].slice.call(database);
```

Comment:
what exact `[].slice.call(database)` does? copies array? Please add comment for dummies like me who dont know what this specific code does